src/Storage/Storage/Blob/Cmdlet/NewAzureStorageContainer.cs

```diff
@@ -27,7 +27,6 @@ namespace Microsoft.WindowsAzure.Commands.Storage.Blob.Cmdlet
     /// create a new azure container
     /// </summary>
     [Cmdlet("New", Azure.Commands.ResourceManager.Common.AzureRMConstants.AzurePrefix + "StorageContainer"),OutputType(typeof(AzureStorageContainer))]
-    [Alias("New-" + Azure.Commands.ResourceManager.Common.AzureRMConstants.AzurePrefix + "DatalakeGen2FileSystem")]
```

Comment:
@blueww confirmed, not alias for previous sprint.

Reply:
Confirm: This alias only on preview module Az.Storage before.